tests/models/sources/test_bigquery_source.py

```diff
@@ -0,0 +1,25 @@
+from visivo.models.sources.bigquery_source import BigQuerySource
+
+
+def test_BigQuerySource_simple_data():
+    data = {
+        "name": "source",
+        "database": "database",
+        "type": "bigquery",
+        "credentials_base64": "base64string", 
+        "project": "project"
+    }
+    source = BigQuerySource(**data)
+    assert source.name == "source"
+
+def test_Source_password_json():
+    data = {
+        "name": "source",
+        "database": "database",
+        "type": "bigquery",
+        "credentials_base64": "base64string", 
+        "project": "project-project"
+    }
+    source = BigQuerySource(**data)
+
+    assert "**********" in source.model_dump_json()
```

Comment:
You probably copied what I had but maybe we could assert that it has the key and then the stars to make sure that the password field is the one that has the stars.

Reply:
Yeah that's a good idea to get more specific.